packages/code-du-travail-frontend/src/outils/indemniteLicenciement/ccn/1979_indemnite.js

```diff
@@ -0,0 +1,35 @@
+import { round } from "../math";
```

Comment:
this file seems unused, should you remove it ?

Reply:
fixed